waku/standards/core/64/network.md

```diff
@@ -17,7 +17,7 @@ useful for generalized messaging, privacy-preserving, scalable and
 accessible even to resource-restricted devices.
 We'll refer to this opinionated deployment simply as
 _the public Waku Network_, _the Waku Network_ or, if the context is clear, _the network_
-in the rest of this document.
+in the rest of this document. All The Waku Network configuration parameters are listed [here](https://github.com/waku-org/nwaku/blob/master/waku/factory/networks_config.nim#L31).
```

Comment:
```suggestion
in the rest of this document.
All The Waku Network configuration parameters are listed [here](https://github.com/waku-org/nwaku/blob/8bfad3ab453f96ac545c7cb0af06d0c0f34d1356/waku/factory/networks_config.nim#L31).
```

Semantic breaks and permalinks for specs.

Reply:
done: fd45bb7495c9123dd2f00becf30643db5660e6e3